tools/src/main/scala/scala/scalanative/build/LLVM.scala

```diff
@@ -96,16 +99,6 @@ private[scalanative] object LLVM {
         config.compilerConfig.targetTriple.map(_ => s"-Wno-override-module")
       multithreadingEnabled ++ usingCppExceptions ++ allowTargetOverrrides
     }
-    val exceptionsHandling = {
-      val targetSpecific = if (isCppRuntimeRequired(config, analysis)) {
-        val opt = if (isCpp) List("-fcxx-exceptions") else Nil
-        List("-fexceptions", "-funwind-tables") ++ opt
```

Comment:
I think these are currently not propagated when compiling .ll files

Reply:
Yes, I think I removed that. Not seeing it.